internal/pipeline/executor.go

```diff
@@ -139,15 +142,32 @@ func (p *pkg) resolveSources() []string {
 			// include only this pkg sources
 			pkgPaths = append(pkgPaths, p.Path())
 		case sourceAllSubPkgs:
-			// TODO(droot): figure out how to enumerate sub packages for current pkgPath.
-			// For now including current pkg
+			// For now including current pkg and subpkgs in current directory
 			pkgPaths = append(pkgPaths, p.Path())
+			files, err := ioutil.ReadDir(p.Path())
+			if err != nil {
+				return nil, err
+			}
+			for _, f := range files {
+				if f.IsDir() {
+					// A directory is a package if it has a Kptfile.
+					// This may change as the concept of a package is expanded such that
+					// every directory is its own kpt package
+					absolutePath := path.Join(p.Path(), f.Name())
+					subPaths, err := pathutil.DirsWithFile(absolutePath, "Kptfile", false)
+					if err != nil {
+						return nil, err
+					}
+					sort.Strings(subPaths)
```

Comment:
We do we sort the paths here? In particular, why do we need them in alphabetical order?

Reply:
This was a discussion I had with @droot. There's no strong reason to sort the subpaths apart from having deterministic output to make testing easier